packages/synapse-react-client/src/components/SynapseTable/SynapseTable.tsx

```diff
@@ -440,7 +440,9 @@ export class SynapseTable extends React.Component<
       this.allRowsHaveId()
 
     const isShowingAddToV2DownloadListColumn: boolean = !!(
-      rowsAreDownloadable && !this.props.hideDownload
+      rowsAreDownloadable &&
+      !this.props.hideDownload &&
+      !isRowSelectionVisible
```

Comment:
It's redundant to show the Add To Download List table column if row selection is available (since it provides the same functionality in the RowSelectionControls if rows are downloadable)